source/frontend/src/models/api/CompensationRequisition.ts

```diff
@@ -8,11 +8,14 @@ import { Api_CompensationFinancial } from './CompensationFinancial';
 import { Api_ConcurrentVersion } from './ConcurrentVersion';
 import { Api_FinancialCode } from './FinancialCode';
 import { Api_InterestHolder } from './InterestHolder';
+import { Api_Project } from './Project';
 
 export interface Api_CompensationRequisition extends Api_ConcurrentVersion, Api_AuditFields {
   id: number | null;
   acquisitionFileId: number;
   acquisitionFile: Api_AcquisitionFile | null;
+  alternateProject?: Api_Project;
```

Comment:
@buddy326 I am a little concerned that this PR seems to primarily interact with undefined instead of null, but then the model here is using a mixture of null and undefined. Can you please confirm if the api is sending undefined or null when the alternate project is not set and update the typing here and all of the checks as appropriate.

Reply:
In general new controllers should be returning null instead of undefined - not sure where the compensation controllers fall on that issue.

Reply:
Ok I will take a look, thank you!